packages/@sanity/mutator/test/parse.test.ts

```diff
@@ -183,6 +183,10 @@ const cases = {
       },
     ],
   },
+  trueOrFalseField: {
```

Comment:
should we add a test that fails if this name is just `true` or `false` to make sure we didn't just break something?

Reply:
Added explicit tests for parsing symbols versus attributes/identifiers